tests/test_unified_scoring_integration.py

```
@@ -0,0 +1,313 @@
+            # Mock evaluation calls
+            mock_evaluator.evaluate_population = AsyncMock(
+                side_effect=[gen0_fitness, gen1_fitness]
+            )
+            
+            # Mock diversity calculation
+            mock_evaluator.calculate_population_diversity = AsyncMock(
+                return_value=0.8
+            )
+            
+            # Create GA and run evolution
+            ga = GeneticAlgorithm(llm_provider=mock_provider)
```

Ruff (0.12.2)

Local variable `ga` is assigned to but never used

Remove assignment to unused variable `ga`

(F841)